src/OpenTelemetry.Exporter.Geneva/Internal/MsgPack/MsgPackTraceExporter.cs

```diff
@@ -32,14 +35,36 @@ internal sealed class MsgPackTraceExporter : MsgPackExporter, IDisposable
         ["messaging.url"] = "messagingUrl",
     };
 
+    internal static readonly Dictionary<string, int> CS40_PART_B_HTTPURL_MAPPING_DICTIONARY = new()
+    {
+        // Mapping from unstable HTTP semconv to httpUrl
+        // Combination of http.scheme, net.host.name, net.host.port, and http.target attributes for HTTP server spans
+        ["http.scheme"] = 0,
+        ["net.host.name"] = 1,
+        ["net.host.port"] = 2,
+        ["http.target"] = 3,
+
+        // Mapping from stable HTTP semconv to httpUrl
+        // Combination of url.scheme, server.address, server.port, url.path and url.query attributes for HTTP server spans
```

Comment:
We could add `http.request.method` as 0th entry. Only if 0th entry has data we could build the URL.